cli/generators/crud/templates/appService.js

```diff
@@ -0,0 +1,81 @@
+import fetch from 'isomorphic-fetch';
+import * as endpoints from './apiEndpoints';
+
+class <%= ucName %>Service {
+  static load<%= pluralizedUcName %>() {
+    const request = new Request(`${endpoints.BASE_URL}${endpoints.GET_<%= pluralizedName.toUpperCase() %>}`, {
+      method: 'GET'
+    });
+
+    return fetch(request).then(response => {
+      return response.json();
+    }).catch(error => {
+      return error;
+    });
```

Comment:
I agree with the service returning data already processed, so I think this is the one that should call `.json()`